pkg/controller/allowlist/allowlist_controller.go

```diff
@@ -36,21 +38,15 @@ import (
 )
 
 const (
-	allowlistDsName      = "cni-sysctl-allowlist-ds"
-	allowlistAnnotation  = "app=cni-sysctl-allowlist-ds"
-	manifestDir          = "../../bindata/allowlist/daemonset"
-	allowlistManifestDir = "../../bindata/network/multus/004-sysctl-configmap.yaml"
+	dsName       = "cni-sysctl-allowlist-ds"
+	dsAnnotation = "app=cni-sysctl-allowlist-ds"
+	dsManifest   = "../../bindata/allowlist/daemonset/daemonset.yaml"
```

Comment:
Can you explain the motivation of this change? I am missing it from the commit message. This is being passed to a `renderDir` function so it wouldn't naturally make sense.

Reply:
I am not sure I agree. The daemonset is represented by the contents of `bindata/allowlist/daemonset/`; the fact that this is represented by a single or multiple manifests is a detail that is best left out from the controller code. Otherwise, if in the future we want to add more manifests (let's say a config map) then we would need to change the code. Rendering directories is the common pattern we follow in CNO to abstract the code from this fact.

Reply:
no strong opinion, i changed to `dsManifestDir = "../../bindata/allowlist/daemonset"`